travel/travel.py

```diff
@@ -25,6 +25,17 @@
 from .res_config import get_basic_passenger_limit
 
 
+def _get_travel_states(self, cr, uid, ids=None, context=None):
```

Comment:
Well, I don't think so. If you put a related field, it's not logical to need again the selection list, but nevertheless, it's not a blocking one. It's only for tracing the problem and if in the future is solved (I think so that it's in v8), you can remove this part